src/style.css

```diff
@@ -1,3 +1,61 @@
+@import url("https://fonts.googleapis.com/css?family=Roboto:500,700");
+
+* {
+  box-sizing: border-box;
+}
+
 body {
+  text-align: center;
   margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
```

Comment:
You need to add styles for your logo and add CSS properties according to design in figma, at the moment your first two tests fails because your logo has wrong styles, you already have a class selector so now you should add it too CSS file and make it